code/style.css

```diff
@@ -1,4 +1,182 @@
-html {
-  background: #0025ff;
+body {
+  background-color: rgb(38, 77, 148);
   color: #fff;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  margin: 0;
+}
+
+/***HEADER STYLING ***/
+
+header {
+  background-color: rgb(128, 38, 83);
+  height: 80px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 16px;
+}
+
+nav {
+display: none;
+ 
+}
+
+nav a {
+  color: #fff;
+  text-decoration: none;
+  margin-left: 16px;
+  font-size: 20px;
+}
+
+.hamburger {
+  display: flex;
+  width: 30px;
+  height: 30px;
+  flex-direction: column;
+  justify-content: space-between;
+  margin-right: 24px;
+  
+}
+
+.hamburger div {
+  width: 30px;
+  height: 5px;
+  background: #fff;
+  border-radius: 5px;
+}
+
+header img {
+width: 50px;
+height: 50px;
```

Comment:
And just some indentation here too, besides here and the nav the CSS looks just as it should!